doc/external-libs/trilinos.html

```diff
@@ -87,6 +88,7 @@ <h5>Installing Trilinos</h5>
     -DTrilinos_ENABLE_MueLu=ON                       \
```

Comment:
can you update the path in line 77 to "-12.10.1"?

Reply:
I did something similar.